<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Consulta de pagos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tPagos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nº Movimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripcion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Importe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apadrinamiento</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>Cuota mes enero</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>2019-01-01</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>35 €</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lSocio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Socio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eSocioId">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eSocioNombre">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>411</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bAtras">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bConsulta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar Pago</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
